src/pages/components/menu-buttons/style.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: Menu buttons
+tabs: ['Usage', 'Style', 'Code', 'Accessibility']
+---
+
+## Color
```

Comment:
Again, wondering if we should start using Aubrey's new style content template here or if you want to do that at a later date.